### PR TITLE
Longevity backup trigger

### PR DESCRIPTION
**Pull request by Rohit-PX**
**What this PR does / why we need it**:
Based off of an earlier PR: https://github.com/portworx/torpedo/pull/591

**Which issue(s) this PR fixes** (optional)

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by Rohit-PX** [comment]
Ran torpedo dev-test here: https://jenkins.portworx.dev/job/Torpedo/job/tp-dev-test/965/

**Comment by Rohit-PX** [comment]
Torpedo dev test job after vendoring.

**Comment by Rohit-PX** [comment]
Ran tests at: https://jenkins.portworx.dev/job/Stork/view/Stork%202.8/job/stork-2.8-px-2.9-dev-k8s-1-21/36/console

File: drivers/backup/auth.go [hunk 1]
```diff
@@ -29,7 +29,8 @@ const (
 	// PxCentralAdminSecretNamespace namespace of PxCentralAdminSecretName
 	PxCentralAdminSecretNamespace = "px-backup"
 	// keycloakEndPoint Endpoint for keycloak
-	keycloakEndPoint = "pxcentral-keycloak-http:80"
+	// TODO: make the namespace configurable for service
+	keycloakEndPoint = "pxcentral-keycloak-http.px-backup.svc.cluster.local:80"
```
Comment (diptiranjanpx) [inline drivers/backup/auth.go:33]: For now it is ok, but if the namespace name changes the endpoint will change . Also to access the endpoint via px-central-endpoint/auth is much better as there will be no dependancy on which ns the test runs.

Lets create a separe ticket for this.

**Review by diptiranjanpx — APPROVED** [review]

**Pull request merged**